src/Illuminate/Bus/Dispatcher.php

```diff
@@ -190,13 +190,36 @@ protected function pushCommandToQueue($queue, $command)
      * @param  array  $pipes
      * @return $this
      */
-    public function pipeThrough(array $pipes)
+    public function setPipes(array $pipes)
     {
         $this->pipes = $pipes;
 
         return $this;
     }
 
+    /**
+     * Get the pipes through which commands should be piped before dispatching.
+     *
+     * @return void
+     */
+    public function getPipes()
+    {
+        return $this->pipes;
+    }
+
+    /**
+     * Add a pipe through which commands should be piped before dispatching.
+     *
+     * @param mixed $pipe
```

Comment:
Is there some type for `$pipe`? Laravel uses simple arrays for typehints instead of `object[]` so it's hard to tell.

Perhaps the type should be `object`?